status/StatusUpdateReporter.py

```diff
@@ -156,12 +167,14 @@ def generateDailyReport(self):
 
             totalMembers = log.members.count()
             didNotSendCount = log.didNotSend.count()
-            sendCount = totalMembers - didNotSendCount
+            invalidUpdatesCount = log.invalidUpdates.count()
+            sendCount = totalMembers - didNotSendCount - invalidUpdatesCount
```

Comment:
```suggestion
            sendCount = totalMembers - (didNotSendCount + invalidUpdatesCount)
```

Isn't this convention better